solutions/cses-1619.mdx

```diff
@@ -184,6 +188,72 @@ int main(){
 }
 ```
 
+</CPPSection>
+	
+<JavaSection>
+	
+<Warning>
+Java solutions using this implementation will TLE on test case #6.
+</Warning>
+
```

Comment:
java solutions are able to pass: [1](https://cses.fi/problemset/hack/1619/entry/807062/) [2](https://cses.fi/problemset/hack/1619/entry/804200/)